node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
```

Comment:
Yes.

---
In reply to: [71904835](https://github.com/opent2t/opent2t/pull/1#discussion_r71904835) [](ancestors = 71904835,71904732)